tests/test_contrib_cern_openid_rest.py

```diff
@@ -196,3 +199,85 @@ def test_account_info_not_allowed_account(app_rest, example_cern_openid_rest):
         "code": 400,
     }
     check_response_redirect_url_args(resp, expected_url_args)
+
+
+def test_identity_changed(app_rest, example_cern_openid_rest, models_fixture):
```

Comment:
Wow... not easy to understand this test... a few tips:
* could you maybe wrap in a function the init/preparation of the test, e.g.
```
def test():
    def _init(): 
         ....
    init()
    ...real test code...
```
* I think that the main thing would be to test if the extra metadata has been fetched or not. I think that I would expect to see:

```
def test():
    ...
    def _test_login_token_no_remote_account():
        assert login succeed
        assert no extra metadata

     def _test_login_token_with_remote_account():
        assert login succeed
        assert extra metadata
```

Reply:
fixed